libs/dal/src/repositories/notification-template/notification-template.entity.ts

```diff
@@ -127,6 +128,8 @@ export class StepVariantEntity implements IStepVariant {
   shouldStopOnFail?: boolean;
 
   bridgeUrl?: string;
+
+  controlVariables?: ControlsDto;
```

Comment:
called as `controlVariables` and not `controls` because we already have the `controls` on the step which is the step schema.